web/themes/custom/pineapple/pineapple.theme

```diff
@@ -49,6 +49,17 @@ function pineapple_theme_suggestions_input_alter(array &$suggestions, array $var
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_user_alter().
+ */
+function pineapple_theme_suggestions_user_alter(array &$suggestions, array $variables) {
+  $user = $variables['elements']['#user'];
+  $roles = $user->getRoles();
+  foreach ($roles as $role) {
+    $suggestions[] = 'user__' . $variables['elements']['#view_mode']. '__' . $role;
+  }
+}
```

Comment:
Suggestion should be in the following pattern: 'user--account--full' where user the bundle name and full is the view mode. I don't think there's a need to create templates based on roles just yet.

Reply:
Okay, done.